.github/workflows/daily-version-check.yml

```diff
@@ -0,0 +1,78 @@
+name: Daily Released Version Check
+
+on:
+  schedule:
+    # Runs at 02:00 UTC every day
+    - cron: '0 2 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  # When triggered manually, it will run for ALL branches defined in the matrix below.
+  workflow_dispatch:
+
+jobs:
+  check_released_versions:
+    name: Check Branch ${{ matrix.target_branch }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+
+    strategy:
+      # fail-fast: false ensures that if one branch check fails,
+      # the checks for other branches will still run to completion.
+      fail-fast: false
+      matrix:
+        target_branch:
+          # The list needs to be adjusted based on the branch where the workflow runs
+          - 'release-v2.8'
+          - 'release-v2.9'
+          - 'release-v2.10'
+          - 'release-v2.11'
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set up Dependencies (jq, curl)
+        run: |
+          echo "Installing jq and curl..."
+          sudo apt-get update -qq
+          sudo apt-get install -y -qq jq curl
+
+      - name: Make Script Executable
+        run: chmod +x ./scripts/check-versions.sh
```

Comment:
Is there a reason this script isn't executable by default?

Reply:
The script is executable by default, and this line is added to ensure the proper permission. 
The line is removed as it is not needed.

Reply:
New run: https://github.com/jiaqiluo/kontainer-driver-metadata/actions/runs/14602880552